src/lib/models/ConfigModel.ts

```diff
@@ -114,7 +114,9 @@ abstract class YamlFormatter {
 			this.toYamlRuleOverrides() + '\n' +
 			'\n' +
 			this.toYamlComment(topLevelDescription.fieldDescriptions!.engines) + '\n' +
-			this.toYamlEngineOverrides() + '\n';
+			this.toYamlEngineOverrides() + '\n' +
+			'\n' +
+			this.toYamlSectionHeadingComment(getMessage(BundleName.ConfigModel, 'template.common.end-of-config')) + '\n';
```

Comment:
I think this end of config message might end up in the file as well. I was hoping it would just be for the terminal output only.